MdePkg/Library/BasePanicLibNull/BasePanicLibNull.uni

```diff
@@ -0,0 +1,16 @@
+// /** @file
+// Panic Library with empty functions.
+//
+// Panic Library with empty functions.
+//
+// Copyright (c) 2023, Microsoft Corporation. All rights reserved.<BR>

```

Comment:
I think that all the license text should just be:

```suggestion
// Copyright (c) Microsoft Corporation.

```

But you can confirm.

Reply:
I forgot to update these before publishing the PR.  Thanks for pointing it out.